contributor-guide/modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,6 @@
 
 * xref:getting-involved.adoc[]
-* xref:tweeting-boost.adoc[]
+* xref:tweeting.adoc[]
```

Comment:
I don't know if this was on purpose but I find this position in the nav hierarchy extremely bad.

<img width="124" alt="image" src="https://github.com/cppalliance/site-docs/assets/5369819/5f07d46b-46b8-43b7-9fd2-809673fcf84e">

It's very unlikely that the first thing a person learning about how to contribute to Boost wants to learn is how people authorized to tweet will tweet.

I thought that would be last and included as a level 2 section of some level 1 section about "branding", "social" or something like that. 

The way it is, it's completely unrelated to any other information in this documentation component, which is all about code.

Reply:
To match the User Guide, I suggest a top level node, at the end of the TOC:
Community
    Tweeting

Reply:
"Community" is a great name.

Reply:
@PeterTurcan if I'm understanding you right, you mean one top-level "Community" with just one child "Tweeting"?

Reply:
Yes. Just one child but we're going to have more pages that can go there. There's stuff related to the ML and there will be a lot of stuff related to the new website.